definitions/ext-ddi/golden_metrics.yml

```diff
@@ -1,10 +1,17 @@
 cpuUtilization:
   title: CPU utilization (%)
   unit: PERCENTAGE
-  query:
-    select: average(kentik.snmp.CPU)
-    from: Metric
-    where: "provider = 'kentik-ipam'"
+  queries:
+    # Infoblox
+    kentik/infoblox-ipam:
+      select: average(kentik.snmp.hrProcessorLoad)
+      from: Metric
+      where: "provider = 'kentik-ipam'"
+    # Cisco Firepower firewalls
+    kentik:
```

Comment:
can you set this one as the first one? it will be the "default" when nothing else matches.

Reply:
@Mesverrum any updates?

Reply:
I've been on vacation.  fixing it now